src/main/java/ogc/rs/database/DatabaseServiceImpl.java

```diff
@@ -998,7 +998,7 @@ public Future<JsonObject> updateStacCollection(JsonObject requestBody) {
         String id = requestBody.getString("id");
         String title = requestBody.getString("title");
         String description = requestBody.getString("description");
-        String datetimeKey = requestBody.getString("datetimeKey");
+        String datetime = requestBody.getString("datetime");
```

Comment:
We can remove this line. The `datetimeKey` is only visible to the server. The provider has no idea of what this key is. So we don't need an update.